packages/slice-machine/src/tracker.ts

```diff
@@ -70,14 +70,17 @@ export class SMTracker {
       .catch(() => console.warn(`Couldn't report identify: Tracking error`));
   }
 
-  async #group(attributes: Record<string, unknown> = {}): Promise<void> {
+  async #group(
+    groupId: string,
+    attributes: Record<string, unknown> = {}
+  ): Promise<void> {
     if (!this.#isTrackingPossible(this.#client)) {
       return;
     }
 
     return this.#client
       .then((client): void => {
-        void client.group(attributes);
+        void client.group(groupId, attributes);
```

Comment:
funny thing, the group function of the SDK takes "UserParameter" for the group call.
Guess it means that groups are considered as Users.